.github/workflows/deploy.yml

```diff
@@ -173,20 +173,24 @@ jobs:
         with:
           ref: master
 
-      - name: Setup Hugo
-        uses: peaceiris/actions-hugo@v2
+      - name: Setup Ruby
+        uses: actions/setup-ruby@v1
         with:
-          hugo-version: '0.79.1'
+          ruby-version: ${{ env.RUBY_VERSION }}
+
+      - name: Install dependencies
+        run: bundle install --jobs=3 --retry=3
+        working-directory: web
 
-      - name: Build via Hugo
-        run: hugo -v --ignoreCache
+      - name: Build site with nano
+        run: bundle exec nanoc
         working-directory: web
 
       - name: Upload web
         uses: actions/upload-artifact@v2
         with:
           name: foreman-docs-web-master
-          path: web/public/
+          path: web/output/
```

Comment:
Why change this here?

Reply:
Technically not needed, but this is a temporary directory anyway. It gets moved to the final destination, no problem I guess. If you insist, I can change nanoc configuration to output to public.

Reply:
No, it's ok. Thanks for explaning!